api/models.py

```diff
@@ -216,35 +223,57 @@ def has_answer_image_url(self):
 
     @property
     def like_count_agg(self):
-        return self.feedbacks.liked().count()
+        return self.like_count + self.feedbacks.liked().count()
 
     @property
     def dislike_count_agg(self):
-        return self.feedbacks.disliked().count()
+        return self.dislike_count + self.feedbacks.disliked().count()
 
     @property
-    def answer_count(self):
-        return self.stats.count()
+    def answer_count_agg(self):
+        return self.answer_count + self.stats.count()
 
     @property
-    def answer_success_count(self):
-        return self.stats.filter(choice=self.answer_correct).count()
+    def answer_success_count_agg(self):
+        return (
+            self.answer_success_count
+            + self.stats.filter(choice=self.answer_correct).count()
+        )
 
     @property
     def answer_success_rate(self):
         return (
             0
-            if (self.answer_count == 0)
-            else int((self.answer_success_count / self.answer_count) * 100)
+            if (self.answer_count_agg == 0)
+            else int((self.answer_success_count_agg / self.answer_count_agg) * 100)
         )
 
     # Admin
     tags_list_string.fget.short_description = "Tag(s)"
+    answer_count_agg.fget.short_description = "# Rép"
+    answer_success_count_agg.fget.short_description = "# Rép Corr"
+    answer_success_rate.fget.short_description = "% Rép Corr"
     like_count_agg.fget.short_description = "# Like"
     dislike_count_agg.fget.short_description = "# Dislike"
-    answer_count.fget.short_description = "# Rép"
-    answer_success_count.fget.short_description = "# Rép Corr"
-    answer_success_rate.fget.short_description = "% Rép Corr"
+
+
+class QuestionStat(models.Model):
```

Comment:
I'd name this `AnswerEvent` but it's just me, to show that :
- this is an event that come in a stream of `created_at`
- this is a single data point, and not a "stat" (stat suggest an aggregation of sorts)

Reply:
t'as raison, je fais la modif